src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(num) {
+      calculator.result += num;
+
+      return calculator;
+    },
```

Comment:
This method shouldn't have access to `result`, assigning to `result` should be in `operate` method
Let's pass two parameters here and return result of operation. Then in `operate` we'll assign result of operation to `result`